geode-junit/src/main/java/org/apache/geode/test/dunit/standalone/VersionManager.java

```diff
@@ -136,8 +138,16 @@ private void findVersions(String fileName) {
     readVersionsFile(fileName, (version, path) -> {
       Optional<String> parsedVersion = parseVersion(version);
       if (parsedVersion.isPresent()) {
-        classPaths.put(parsedVersion.get(), path);
-        testVersions.add(parsedVersion.get());
+        if (parsedVersion.get().equals("140") && SystemUtils.isJavaVersionAtLeast(900)) {
```

Comment:
Is this worth some discussion on the `dev` list?  I would think that any pre `1.8` or `1.9` version of Geode shouldn't be expected to support Java 9+.  Even if this is all it takes to pass the tests as they currently stand, it feels like a suckerpunch waiting to happen to run tests that aren't validly configured.

Reply:
Let me ask the question in dev list.

Reply:
since not much discussion about this on the dev list, looks like people don't care much if we go either way. Can we do this for now? We can always reevaluate if further upgrade tests fail on 9+ and get rid of all old versions.